src/cmdstan-guide/installation.qmd

```diff
@@ -399,93 +387,38 @@ xcode-select --install
 #### Windows {#windows}
 
 
-The Windows toolchain consists of programs `g++`, the C++ compiler, and `mingw32-make`, the GNU Make utility.
+The Windows toolchain consists of programs `g++`, the C++ compiler, and `make`, the GNU Make utility.
 To check if these are present,
 open a command shell [^1]
 and type:
 ```
 g++ --version
-mingw32-make --version
+make --version
 ```
 
-CmdStan is known compatible with the RTools40, RTools42, and RTools43 toolchains. These require slightly
-different steps to configure, so please follow the appropriate steps below. All toolchains will require
+CmdStan is known compatible with the RTools43 toolchain. The toolchais will require
 updating your `PATH` variable, See [these instructions](https://helpdeskgeek.com/windows-10/add-windows-path-environment-variable/)
 for details on changing the `PATH` if you are unfamiliar. The following instructions will assume that the default installation
 directory was used, so be sure to update the paths accordingly if you have chosen a different
 directory.
 
-##### RTools40
 
-RTools40 provides both a standard `g++-8` toolchain and a `g++-10` Universal C Runtime (UCRT) toolchain.
-Note the that newer `g++-10` UCRT toolchain is only available for 64-bit systems, whereas the `g++-8`
-toolchain is available for both. Additionally the UCRT is only natively supported on Windows 10 and newer,
-older systems will require a [Microsoft update](https://support.microsoft.com/en-us/topic/update-for-universal-c-runtime-in-windows-c0514201-7fe6-95a3-b0a5-287930f3560c)
+##### RTools43
 
-###### Installation
-
-Download the [installer](https://github.com/r-windows/rtools-installer/releases/download/2022-02-06/rtools40-x86_64.exe)
-and complete the prompts to install.
-
-Next, you need to add the location of the toolchain to your `PATH` environment variable so that it
-can be called from the command line. Add the following lines to your `PATH`:
-```
-C:\rtools40\usr\bin
-
-# Add only one of the below
-C:\rtools40\mingw32\bin   # 32-bit g++-8
-C:\rtools40\mingw64\bin   # 64-bit g++-8
-C:\rtools40\ucrt64\bin    # 64-bit g++-10 (UCRT)
-```
-
-CmdStan additionally needs the `mingw32-make` utility, which you can install using RTools.
-Navigate to the installation directory (e.g., `C:\rtools40`) and launch the `msys2.exe` file.
-Execute the appropriate command below to install `mingw32-make` for your selected toolchain:
-```
-pacman -Sy mingw-w64-i686-make            # 32-bit g++-8
-pacman -Sy mingw-w64-x86_64-make          # 64-bit g++-8
-pacman -Sy mingw-w64-ucrt-x86_64-make     # 64-bit g++-10 (UCRT)
-```
-
-##### RTools42 & RTools43
-
-Both RTools42 & RTools43 provide 64-bit UCRT toolchains, where RTools42 provides g++-10 and
-RTools43 provides g++-12. The installation/configuration is identical for both toolchains.
+RTools43 provides a 64-bit UCRT toolchains with GCC 12.3.
```

Comment:
I think only describing rtools43 is fine. This is probably a bit too short and technical to be useful to most users, though.

1. Are we sure that the `x86_64-w64-mingw32.static.posix` folder even is UCRT based?
2. If it is, we should probably retain the note that the UCRT requires windows 10+?

Reply:
Yep it's definitely UCRT, [mentioned in the toolchain doc](https://cran.r-project.org/bin/windows/Rtools/rtools43/rtools.html).

Good point on keeping the Windows 10 note, didn't mean to delete that.

Any thoughts on changes for making it less technical? I'm not sure what would be useful here

Reply:
The best thing might be to say something like "RTools 43 provides all of the necessary tools to build Stan, including a C++ compiler and the `make` tool" and then link to those docs for more specifics?

Reply:
Good shout, will do